Formula/prestodb.rb

```diff
@@ -13,6 +13,8 @@ class Presto < Formula
     sha256 "e7b889a84cbad4dc7fc2f920e37d831cffcbb6bfd67c2f0878d6a5f6cfee02dc"
   end
 
+  conflicts_with "prestosql", :because => "both install `presto` and `presto-server` binaries"
```

Comment:
Move this as per-`brew style`'s output. `brew style --fix` should do it for you automatically.

Reply:
@MikeMcQuaid `brew style` didn't output anything for either formula. However, looking at other formula, it seems to come after `depends_on`, so I moved it there. Please let me know if it should be somewhere else.